react-app-drawer-component/my-app/src/App.js

```diff
@@ -0,0 +1,14 @@
+import MainPage from "./MainPage";
+import './styles.css';
+
+const headings= ['Menu', 'About', 'Get Started', 'Sign In', 'Contact Us']
```

Comment:
This isn't formatted correctly. Do you have prettier set up?

Reply:
I do have prettier set up, I wonder why it didn't catch this